### PR TITLE
Service closure integration branch

### PR DESCRIPTION
**Pull request by jits**
~~DO NOT MERGE~~

### CODE REVIEW

File: app/views/devices_guidance/index.html.erb [hunk 1]
```diff
@@ -13,49 +13,36 @@
...
     </ul>
 
     <h2 class="govuk-heading-m">
-      Managing devices
+      About the devices provided by DfE
     </h2>
 
     <ul class="govuk-list govuk-list--spaced govuk-!-margin-bottom-7">
-      <%= render partial: 'page_link_and_description', collection: @manage_devices_pages, as: :page %>
+      <%= render partial: 'page_link_and_description', collection: @before_you_order_pages, as: :page %>
```
Comment (Lloyd-K) [inline app/views/devices_guidance/index.html.erb:45]: Please apply text from the 'Laptops and tablets (index page)' Master

**Review by Lloyd-K — CHANGES_REQUESTED** [review]

File: app/views/devices_guidance/about_the_offer.md [hunk 2]
```diff
@@ -1,42 +1,34 @@
-## Ordering devices for schools, colleges, academy trusts and local authorities from September 2020
+## Devices for schools, colleges, academy trusts and local authorities from September 2020 to June 2021
 
-DfE is providing support to help disadvantaged children and young people who are otherwise unable to access remote education.
+DfE is provided support to help disadvantaged children and young people who were otherwise unable to access remote education.
```
[the remark this answers is not in the document]
Comment (kenfodder) [reply app/views/devices_guidance/about_the_offer.md]: Done

File: app/views/devices_guidance/device_specification.md [hunk 3]
```diff
@@ -40,53 +22,30 @@ All devices provided before September 2020 were delivered with safeguarding and
...
+* it was visually inspected, and where possible any labels (such as asset tags) were removed
+* it switched on and powered up
+* the screen, CPU, RAM and HDD were tested
+* the device was reset
 
-## iPads and Windows tablets
+These devices met the minimum technical specifications for remote education. They may have residual asset labels, external scratches, smart water application, etching or writing on the exterior of the device.
 
-Apple iPads and Microsoft Windows tablets are now permanently out of stock. See our guide for information on [setting up and managing tablets you've already received](/devices/preparing-microsoft-windows-laptops-and-tablets).
+If devices came with any remaining manufacturer's warranty (this was less than a year).
```
[the remark this answers is not in the document]
Comment (kenfodder) [reply app/views/devices_guidance/device_specification.md:48]: Done

File: app/views/devices_guidance/about_the_offer.md [hunk 4]
```diff
@@ -1,42 +1,34 @@
...
-* in years 3 to 11 with [state-funded places in independent special schools or alternative provision](/devices/how-to-order-laptops-for-independent-special-schools), who are eligible for free school meals
-
-[Find out how we’ve calculated how many devices you can order](/devices/allocation-and-specification).
-
-[Find out how to order laptops and tablets](/devices/how-to-order) and [get the internet](/internet-access).
-
-## Devices for social care leavers
-
-Local authorities can [find out how to order laptops and get internet access for eligible care leavers](/devices/how-to-order-laptops-for-social-care-leavers).
+[Find out how we calculated how many devices could be ordered](/devices/allocation-and-specification).
```
[the remark this answers is not in the document]
Comment (kenfodder) [reply app/views/devices_guidance/about_the_offer.md:25]: Done

File: app/views/devices_guidance/device_allocations.md [hunk 5]
```diff
@@ -1,67 +1,32 @@
...
 
-[Contact us](https://get-help-with-tech.education.gov.uk/get-support) to make a request. 
+Over 220,000 laptops and tablets were made available to distribute as part of the [first phase of the Get help with technology programme](https://www.gov.uk/guidance/laptops-tablets-and-4g-wireless-routers-provided-during-coronavirus-covid-19). The most disadvantaged and vulnerable young people and children were prioritised to make sure they can access social care and remote education.
 
-Do not submit any personally identifiable information about pupils that we do not need to process your support query. If we receive a request that contains such information, we’ll ask you to resend it.
+The DfE considered all allocation queries raised by local authorities and academy trusts, and reviewed the submitted evidence against the initial allocation. DfE then compared those queries to similar requests received nationwide, and balanced these against the total number of devices available. More devices have been provided where there was an evidenced need to support eligible children.
 
+## How devices were allocated for care leavers in 2021
```
[the remark this answers is not in the document]
Comment (kenfodder) [reply app/views/devices_guidance/device_allocations.md:25]: Done

File: app/views/devices_guidance/index.html.erb (hunk 1 above)
Comment (kenfodder) [reply app/views/devices_guidance/index.html.erb:45]: Done

File: app/views/devices_guidance/device_specification.md [hunk 6]
```diff
@@ -1,36 +1,18 @@
...
-* it switches on and powers up
-* the screen, CPU, RAM and HDD are tested
-* the device is reset
-
-These devices will meet the minimum technical specifications for remote education. They may have residual asset labels, external scratches, smart water application, etching or writing on the exterior of the device.
-
-If devices have a remaining manufacturer's warranty, it will be for less than a year. 
-
+<div class="govuk-inset-text">
+We’re now permanently out of stock of all laptops, tablets and routers.
```
[the remark this answers is not in the document]
Comment (kenfodder) [reply app/views/devices_guidance/device_specification.md]: Done

File: app/views/pages/what_to_do_if_you_cannot_get_laptops_tablets_or_internet_access_from_dfe.html.erb [hunk 7]
```diff
@@ -21,22 +21,17 @@
...
-        <%= govuk_link_to 'nurseries and infant schools', what_to_do_if_you_cannot_get_laptops_tablets_or_internet_access_from_dfe_path(anchor:'nurseries-and-infant-schools') %>
-      </li>
-      <li>
-        <%= govuk_link_to 'independent schools', what_to_do_if_you_cannot_get_laptops_tablets_or_internet_access_from_dfe_path(anchor:'independent-schools') %>
-      </li>
-      <li>
-        <%= govuk_link_to 'schools, colleges and further education institutions in Northern Ireland, Scotland and Wales', what_to_do_if_you_cannot_get_laptops_tablets_or_internet_access_from_dfe_path(anchor:'schools-colleges-and-further-education-institutions-in-northern-ireland-scotland-and-wales') %>
+        <%= govuk_link_to 'educational settings', what_to_do_if_you_cannot_get_laptops_tablets_or_internet_access_from_dfe_path(anchor:'educational-settings') %>
       </li>
     </ul>
```
[the remark this answers is not in the document]
Comment (kenfodder) [reply app/views/pages/what_to_do_if_you_cannot_get_laptops_tablets_or_internet_access_from_dfe.html.erb]: Done

**Review by kenfodder — APPROVED** [review]

**Pull request merged**